accepted/PSR-4-autoloader-examples.md

```diff
@@ -117,11 +118,14 @@ class Psr4AutoloaderClass
     /**
      * Register loader with SPL autoloader stack.
      *
+     * @param bool $prepend If true, prepend the autoloader to the stack
+     * instead of appending it.
      * @return void
      */
-    public function register()
+    public function register($prepend = false)
     {
-        spl_autoload_register(array($this, 'loadClass'));
+        spl_autoload_extensions('.php');
+        spl_autoload_register(array($this, 'loadClass'), false, $prepend);
```

Comment:
second parameter is `false` to omit `@throws` in PHPDoc, this added for static analyze tools